Code/Framework/AzFramework/Platform/Linux/platform_linux.cmake

```diff
@@ -11,10 +11,16 @@
 if (${PAL_TRAIT_LINUX_WINDOW_MANAGER} STREQUAL "xcb")
 
     find_library(XCB_LIBRARY xcb)
+    find_library(XCB_XKB_LIBRARY xcb-xkb)
+    find_library(XKBCOMMON_LIBRARY xkbcommon)
+    find_library(XKBCOMMON_X11_LIBRARY xkbcommon-x11)
 
     set(LY_BUILD_DEPENDENCIES
         PRIVATE
             ${XCB_LIBRARY}
+            ${XKBCOMMON_LIBRARY}
+            ${XKBCOMMON_X11_LIBRARY}
+            ${XCB_XKB_LIBRARY}
```

Comment:
Shouldn't we only set the `PAL_TRAIT_LINUX_WINDOW_MANAGER_XCB` if we actually find all 4 libraries?

Reply:
I'm not sure it matters. If those libraries are not found, they will be set to `XCBCOMMON_LIBRARY-NOTFOUND`, and then CMake will issue an error during project generation that the AzFramework library links to a library that was not found. We could add the `REQUIRED` argument to the `find_library` calls, but either way the project generation will fail.